cwms_radar_api/src/main/java/cwms/radar/data/dto/Tsv.java

```diff
@@ -0,0 +1,88 @@
+package cwms.radar.data.dto;
+
+import java.util.Date;
+import javax.persistence.EmbeddedId;
```

Comment:
What led to requiring these imports?

Reply:
CWMSMobile used similar classes with hibernate - I should have removed them.